index.html

```diff
@@ -271,5 +271,59 @@ <h2 class="categorie-title">
     </ul>
 </section>
 </main>
+<footer class="footer-container" id="footer-container-id">
+  <div class="footer-form">
+    <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+    <form action="https://formspree.io/f/xzbovlzk" method="post">
+      <ul>
+        <li>
+          <input type="text" name="Name" placeholder="Full name">
+        </li>
+        <li>
+          <input type="email" name="Email" placeholder="Email address">
+        </li>
+        <li>
+          <textarea name="Message" id="msg" cols="30" rows="10">Enter text here</textarea>
+        </li>
+        <li class="submit-button">
+          <button type="submit">
+              Get in touch
+          </button><br>
+          <small></small>
+      </li>
+      </ul>
+    </form>
+
+    
+  </div>
+  <ul class="social-media">
+    <li>
+      <a href="">
+        <img src="./img/tweet.svg" alt="twitter icon">
+      </a>
+    </li>
+    <li>
+      <a href="">
+         <img src="./img/lnkdn.svg" alt="linkedin icon">
+      </a>
+    </li>
```

Comment:
- [ ] Kindly use the `flexbox`  CSS property to layout the elements on this section.

Reply:
Thank you very much for your kind response but I have already added the `flex box` CSS property in the `social-media` class. Isn't that okay?

Reply:
Yeah but It will be better if you can also add it to the social-media class or `ul` container

Reply:
@moise10r yeah I had added to the `social-media` class. But now I have added a new class `social-media-footer` and added `flex box` in it too.

Reply:
Great :+1: